examples/pruner/job.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: batch/v2alpha1
+kind: ScheduledJob
+metadata:
+  name: image-prune
+spec:
+  schedule: 0/15 * * * ?
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: prune
+            image: openshift/origin:latest
+            args: [ "admin", "prune", "images", "--confirm"]
+          restartPolicy: OnFailure
```

Comment:
In both cases it will create pods, with Never the pods will be there for ever, with OnFailure, they will get replaced. That's the only difference.